formats/json/commonTest/src/kotlinx/serialization/test/TestingFramework.kt

```diff
@@ -38,10 +38,10 @@ inline fun <reified T : Any> StringFormat.assertStringFormAndRestored(
     assertEquals(original, restored)
 }
 
-inline fun <reified T : Any> assertSerializedAndRestored(
+fun <T : Any> assertSerializedAndRestored(
```

Comment:
Why this change is needed here?

Reply:
well, `inline reified` is not necessary for this function at all, but prevent IDEA debugger from working (AFAIR breakpoints didn't hit properly)

Reply:
Ok then